src/gmail.d.ts

```diff
@@ -1031,14 +1034,14 @@ interface GmailCache {
 ////////////////////////////////////////////////////////////////////////////////
 
 declare class Gmail<T extends string=never> {
-    constructor(localJQuery?: JQueryStatic);
+    constructor(localJQuery?: JQueryStatic | false);
```

Comment:
In my opinion, it would best to make this parameter mandatory ´jQuery | false´

That way, typescript code which were calling new Gmail()
will fail to compile with the new version, instead of throwing the runtime exception.

Reply:
Fixed

Reply:
In that case we would probably need to remove check for global jQuery in code (as we either would provide it or want to disable it) and *that* would break compatibility. 🙂
Or it could be OK to have TypeScript definitions stricter than actual library expectation - not everyone uses TS in the end.